auth-web/src/views/auth/home/HomeView.vue

```diff
@@ -181,10 +181,8 @@ export default class HomeView extends Vue {
     return !!this.currentAccountSettings
   }
 
-  private goToManageBusinesses (isNumberedCompanyRequest: boolean = false): void {
-    let manageBusinessUrl: any = { path: `/${Pages.MAIN}/${this.currentAccountSettings.id}` }
-    if (isNumberedCompanyRequest) manageBusinessUrl.query = { isNumberedCompanyRequest }
-
+  private goToManageBusinesses (): void {
+    let manageBusinessUrl = { path: `/${Pages.MAIN}/${this.currentAccountSettings.id}` }
```

Comment:
We no longer need to load + create a numbered company (there is now a button right on the My Business Registry page), so I cleaned up this code.